highlight_view.py

```diff
@@ -989,4 +1029,382 @@ def sort_by_type(error_type):
             heading=heading,
             messages='<br />'.join(filled_templates)
         )
-    return all_msgs
+    return all_msgs, quick_actions
+
+
+def escape_text(text):
+    # type: (str) -> str
+    return html.escape(text, quote=False).replace(' ', '&nbsp;')
+
+
+from contextlib import contextmanager
+if MYPY:
+    from typing import Iterator, NamedTuple
+    Action = NamedTuple("Action", [("description", str), ("fn", Callable)])
+    TextRange = NamedTuple("TextRange", [("text", str), ("range", sublime.Region)])
+else:
+    from collections import namedtuple
+    Action = namedtuple("Action", "description fn")
+    TextRange = namedtuple("TextRange", "text range")
+
+
+class sl_fix_by_ignoring(sublime_plugin.TextCommand):
+    def run(self, edit):
+        view = self.view
+        window = view.window()
+        assert window
+        sel = [s for s in view.sel()]
+        if len(sel) > 1:
+            window.status_message("Only one cursor please.")
+            return
+
+        if not sel[0].empty():
+            window.status_message("Only cursors no selections please.")
```

Comment:
I think it's reasonable to take `sel[0].a` even if the selection is a range?

Reply:
A range (over multiple lines) would be a perfect match to suggest "disable for this block". E.g.

```
// disable
... selected block
// enable
```

Otherwise a range should probably only select the "selected" errors. (Currently we take `.a` but actually expand to the whole line.)